src/notation/internal/notationactioncontroller.cpp

```diff
@@ -164,7 +164,7 @@ void NotationActionController::init()
     registerAction("septuplet", [this]() { putTuplet(7); }, &Controller::noteOrRestSelected);
     registerAction("octuplet", [this]() { putTuplet(8); }, &Controller::noteOrRestSelected);
     registerAction("nonuplet", [this]() { putTuplet(9); }, &Controller::noteOrRestSelected);
-    registerAction("custom-tuplet", &Controller::putTuplet);
```

Comment:
Default "enabler" is that the notation window is active, which *isn't* true when this is sent from the tuplet dialog.